src/sqlparser/src/ast/statement.rs

```diff
@@ -295,6 +313,7 @@ impl Parser {
             format,
             row_encode,
             row_options,
+            key_encode: None,
```

Comment:
We share parse `KEY ENCODE` inside this function as well, and the caller can check `is_some()` if that is not supported. (Alternatively, we can also pass an extra `accept_key_encode: bool` parameter.) It is wired to return `None` and let the caller to further mutate it.